cirq/experiments/__init__.py

```diff
@@ -30,6 +30,13 @@
     xeb_fidelity,
 )
 
+from cirq.experiments.random_quantum_circuit import (
```

Comment:
The name of this code needs to be more descriptive. "Random quantum circuit" can mean a lot of things. Make it like "random_rotations_between_interaction_layers" or something. Be as descriptive as possible... although the name should fit on a line.

Reply:
I think we could keep the module name as is or perhaps just add "generation" to it (as in `random_quantum_circuit_generation` or `random_circuit_generation`) with the intention for any future RQC-generation code to be added here as well.

Reply:
I renamed the module to `random_quantum_circuit_generation`

Reply:
@Strilanc you okay with this?